### PR TITLE
chore(stdlib): Convert Bytes and Buffer to @unsafe

### PR DESCRIPTION
**Pull request by ospencer**

### CODE REVIEW

**Review by marcusroberts — APPROVED** [review]
LGTM!

**Review by peblair — APPROVED** [review]
Parfait! 🔥

File: stdlib/buffer.gr [hunk 1]
```diff
@@ -423,8 +358,8 @@ export let rec addString = (string, buffer) => {
  *
  * @since v0.4.0
  */
-@disableGC
-export let rec addStringSlice = (start: Number, length, string, buffer) => {
+@unsafe
+export let addStringSlice = (start: Number, length, string, buffer) => {
```
Comment (phated) [inline stdlib/buffer.gr:362]: Conflicts with @peblair's PR

**Review by phated — APPROVED** [review]
This is so cool! Love it

**Pull request merged**